projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/query-management/components/query-check-your-answers/query-check-your-answers.component.ts

```diff
@@ -0,0 +1,45 @@
+import { Component, EventEmitter, Input, OnInit, Output, ViewEncapsulation } from '@angular/core';
+import { CaseField } from '../../../../../domain';
+import { caseFieldMockData } from '../../__mocks__';
+import { QueryListItem } from '../../models';
+
+@Component({
+  selector: 'ccd-query-check-your-answers',
+  templateUrl: './query-check-your-answers.component.html',
+  styleUrls: ['./query-check-your-answers.component.scss'],
+  encapsulation: ViewEncapsulation.None
+
+})
+export class QueryCheckYourAnswersComponent implements OnInit {
+
```

Comment:
@DavidJayakumar - can remove extra blank line here (the linter will complain about this)

Reply:
done